### PR TITLE
fix: remove default h3 styles on product pages

### PR DESCRIPTION
**Pull request by duynguyen**
Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/products/clone-screening/mammalian-screening/cloneselect-imager
- After: https://fix-product-h3--moleculardevices--hlxsites.hlx.page/products/clone-screening/mammalian-screening/cloneselect-imager

### CODE REVIEW

**Comment by duynguyen** [comment]
@lydiapuric - I need to revert the h3 styles on product pages because it breaks quite many places.

File: styles/styles.css [hunk 1]
```diff
@@ -827,18 +827,12 @@ main .section.wave .default-content-wrapper p:first-of-type {
   margin-bottom: 10px;
 }
 
-.product .section.tabs h2,
-.section.tabs h3  {
```
Comment (lydiapuric) [inline styles/styles.css:831]: There are pages that have tabs but are not product pages, like the About Us page (/about-us)

Comment (mhaack) [reply styles/styles.css:831]: Hm, but should the style depend on the tabs? The it should be better per page theme or template if it is specific about the about is page.

Comment (duynguyen) [reply styles/styles.css:831]: @lydiapuric - are we good here to proceed?

**Comment by lydiapuric** [comment]
@duynguyen Yes, go ahead.

**Review by lydiapuric — APPROVED** [review]

**Pull request merged**